### DISCUSSION TITLE
Main settings don't stick

### DISCUSSION THREAD

**Discussion by Watt56** [dhowe/AdNauseam#2241] [Q&A]
Firefox browser 108.0.1
Windows 10

For the past couple of days I've been noticing a lot of ads.  When I open AdNauseam settings, I get two yellow and one orange banner at the top asking if I want to hide ads, click ads, and block malware.  And, indeed, those three buttons on the Settings tab are shown as Off.  I turn them On.  When I close out of the browser and reopen, the ads are there again, as well as the three AdNauseam banners; i.e., the settings do not stick.  Again, this just started happening.  I **think** there was a Firefox update a day or so ago.

I am aware of the discussion here: https://github.com/dhowe/AdNauseam/discussions/1850.  But I hope I don't have to do an entire Profile reset on Firefox. 

Thanks.

EDIT: I saved AdN settings, deleted AdN from Firefox, then reinstalled it.  So far, it is working again.  So, simple solution.

**Comment by dhowe** [comment]
Thanks @Watt56 - we've noticed that very occasionally the AdNauseam profile is left in a corrupt state on Firefox. We haven't been able to recreate this consistently as yet, but the fix is the one you already found: create a new Firefox profile, as [described here](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)

**Comment by Watt56** [comment]
I didn't actually create a new Firefox Profile.  I just removed AdNauseam and reinstalled it, without touching other Firefox settings.

**Comment by dhowe** [comment]
That should work as well
